icu4j/main/tests/core/src/com/ibm/icu/dev/test/format/RbnfTest.java

```diff
@@ -1839,4 +1839,14 @@ public void testRoundingUnrealNumbers() {
         assertEquals("infinity", rbnf.format(Double.POSITIVE_INFINITY));
         assertEquals("not a number", rbnf.format(Double.NaN));
     }
+
+    @Test
+    public void TestNumberingSystem() {
+        RuleBasedNumberFormat rbnf = new RuleBasedNumberFormat(ULocale.US, RuleBasedNumberFormat.NUMBERING_SYSTEM);
+
+        assertEquals("Wrong result with default rule set", "123", rbnf.format(123));
```

Comment:
This test currently fails because I didn't know how to rebuild icudata.jar with the rule change I made in icu4c/source/data/rbnf/root.txt.  If somebody can clue me in, I'll fix this.

Reply:
Looks like you fixed it?

Reply:
I did fix it.  Thanks.